faker/sphinx/documentor.py

```diff
@@ -104,8 +103,7 @@ def _write_base_provider_docs():
 
 def _write_standard_provider_docs():
     for provider_name in STANDARD_PROVIDER_NAMES:
-        fname = os.path.join(DOCS_ROOT, 'providers', '%s.rst' % provider_name)
-        with open(fname, 'wb') as fh:
+        with (DOCS_ROOT / 'providers' / '{}.rst'.format(provider_name)) as fh:
```

Comment:
We could use an f-string here:

```suggestion
        with (DOCS_ROOT / 'providers' / f'{provider_name}.rst') as fh:
```

Reply:
There are plenty of f-strings there and I waited for the Python 3.6+ merge. I'll do all f-strings in the next step.